packages/documentation/src/stories/foundation/metadata/metadata-example.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head prefix="og: https://ogp.me/ns#">
+    <!--Content Security Policy-->
+    <meta
+      http-equiv="Content-Security-Poilicy"
+      content="default-src self; connect-src https://unpkg.com/"
+    />
+    <!-- some general tags-->
+    <meta charset="utf-8" />
+    <title>Metadata example</title>
+    <meta name="author" content="DesignSystem" />
+    <meta name="keywords" content="metadata, example" />
+    <meta name="description" content="An Examle of how metadata of a webpage could look like." />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!-- search engines -->
+    <meta name="robots" content="noarchive, nofollow" />
+    <!-- google language -->
+    <link rel="alternate" hreflang="en-gb" href="https://en-gb.example.com/page.html" />
+    <link rel="alternate" hreflang="en-us" href="https://en-us.example.com/page.html" />
+    <link rel="alternate" hreflang="en" href="https://en.example.com/page.html" />
+    <link rel="alternate" hreflang="de-de" href="https://de-de.example.com/page.html" />
+    <link rel="alternate" hreflang="de-ch" href="https://de-ch.example.com/page.html" />
+    <link rel="alternate" hreflang="fr" href="https://fr.example.com/page.html" />
+    <link rel="alternate" hreflang="it" href="https://it.example.com/page.html" />
+    <link rel="alternate" hreflang="x-default" href="https://www.example.com/" />
```

Comment:
```suggestion
    <!-- language alternatives -->
    <link rel="alternate" hreflang="en" href="https://en.example.com/page.html" />
    <link rel="alternate" hreflang="de" href="https://de.example.com/page.html" />
    <link rel="alternate" hreflang="fr" href="https://fr.example.com/page.html" />
    <link rel="alternate" hreflang="it" href="https://it.example.com/page.html" />
    <link rel="alternate" hreflang="x-default" href="https://www.example.com/" />
```

Can you explain the `x-default` alternative? It's totally unclear why this is added as an example here.